acs/acs.go

```diff
@@ -28,39 +27,35 @@ type ACS struct {
 	bbaMap map[cleisthenes.Address]*bba.BBA
```

Comment:
How about create new structure like broadcastResult?

Reply:
yes and it is TODO!